libs/server/repository/infrastructure/src/index.ts

```diff
@@ -9,3 +9,5 @@ export * from './lib/bitbucket/repositories/bitbucket-reviewer.repository';
 export * from './lib/github/repositories/github-pr.repository';
 export * from './lib/github/repositories/github-repository.repository';
 export * from './lib/github/repositories/github-reviewer.repository';
+
```

Comment:
You could remove this empty line

Reply:
It's to underline the separation between bitbucket, GitHub and GitLab. There are already such empty lines in the file.